examples/java/src/main/java/org/apache/beam/examples/complete/cdap/PluginWrapperBuilder.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.examples.complete.cdap;
+
+import io.cdap.cdap.api.plugin.PluginConfig;
+
+/**
+ * Class for building {@link PluginWrapper} object.
+ */
+public abstract class PluginWrapperBuilder<F, FP> {
+    protected PluginWrapper<F, FP> pluginWrapper;
+    protected PluginConfig pluginConfig;
+
+    public PluginWrapperBuilder<F, FP> withConfig(PluginWrapper<F, FP> pluginWrapper, PluginConfig pluginConfig) {
+        this.pluginWrapper = pluginWrapper;
+        this.pluginConfig = pluginConfig;
+        return this;
+    }
+
+    public abstract void build();
```

Comment:
According to the Builder pattern, `build()` method should return an instance of the `PluginWrapper` class.

Reply:
Changed return type to `PluginWrapper<F, FP>`.